scripts/train/train.py

```diff
@@ -575,4 +19,4 @@ def main(cfg: DictConfig) -> Trainer:
     cli_cfg = om.from_cli(args_list)
     cfg = om.merge(yaml_cfg, cli_cfg)
     assert isinstance(cfg, DictConfig)
-    main(cfg)
+    train(cfg)
```

Comment:
could we have a helper function called `train_from_yaml` so we can share the code between here and the code in cli.py?